stl/inc/format

```diff
@@ -1376,11 +1392,49 @@ _OutputIt vformat_to(
     return _Handler._Ctx.out();
 }
 
-// FUNCTION vformat
-string vformat(string_view _Fmt, format_args _Args);
-wstring vformat(wstring_view _Fmt, wformat_args _Args);
-string vformat(const locale& _Loc, string_view _Fmt, format_args _Args);
-wstring vformat(const locale& _Loc, wstring_view _Fmt, wformat_args _Args);
+string vformat(string_view _Fmt, format_args _Args) {
+    string _Str;
+    vformat_to(back_inserter(_Str), _Fmt, _Args);

```

Comment:
`vformat_to` and `back_inserter` both need `_STD` qualification. (Several places.)

Reply:
I was under the understanding that there can't be ADL hijacking since these are types within the standard. Is there a scenario when this is unsafe?

Reply:
We conventionally `_STD`-qualify pretty names regardless so it's obvious even to someone passing by and glancing at this code that we didn't forget to qualify. But yes, I agree these arguments are of types that we fully control so ADL hijacking isn't possible here.

Reply:
I can still add the qualifier, I just wanted to make sure my understanding of when `_STD` is required as opposed to when it is stylistic.